android/src/main/kotlin/com/pusher/channels_flutter/PusherChannelsFlutterPlugin.kt

```diff
@@ -190,7 +190,7 @@ class PusherChannelsFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAw
                     mutex.release()
                 }
 
-                override fun error(s: String?, s1: String?, o: Any?) {
+                override fun error(s: String, s1: String?, o: Any?) {
                     // Log.e(TAG, "ERROR: $s $s1 $o")
```

Comment:
Isn't it better to remove this comment as well?

Reply:
done

Reply:
done